internal/provider/resources/block_test.go

```diff
@@ -0,0 +1,39 @@
+package resources_test
+
+// func fixtureAccBlockCreate(name string) string {
```

Comment:
commenting this out for now, as they will fail if we dont register the resource to the provider